### PR TITLE
fix(2985): Migrate Use of “Master” to “Main” in Docs

### PR DESCRIPTION
**Pull request by tjleing**
Courtesy of @oieduardorabelo -- thank you!  We plan on performing this migration tomorrow 😄 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/spellcheck.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@master
+        uses: actions/checkout@main
       - name: Setup Node.js 10.x
-        uses: actions/setup-node@master
+        uses: actions/setup-node@main
```
Comment (tjleing) [inline .github/workflows/spellcheck.yml:11]: @jakeburden GH Actions -- for tomorrow.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -9,9 +9,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@master
+        uses: actions/checkout@main
       - name: Setup Node.js 10.x
-        uses: actions/setup-node@master
+        uses: actions/setup-node@main
```
Comment (tjleing) [inline .github/workflows/test.yml:14]: @jakeburden see above.

File: .github/workflows/spellcheck.yml (hunk 1 above)
Comment (jakeburden) [reply .github/workflows/spellcheck.yml:11]: Thank you!

File: docs/cli/teams/shared.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Share single environment
 description: Learn the recommended workflow for multiple team members sharing a single Amplify environment.
 ---
 
-You have two independent environments (`main` & `dev`) in the cloud and have corresponding Git branches with your Amplify backend infrastructure code on Git. Suppose all team members want to work on the same Amplify project and push backend related changes to the same `dev` environment to test their changes. Each team member would run the following:
+You have two independent environments (main & dev) in the cloud and have corresponding git branches with your amplify backend infrastructure code on Git. Suppose all team members want to work on the same Amplify project and push backend related changes to the same dev environment to test their changes. Each team member would run the following:
```
Comment (tjleing) [inline docs/cli/teams/shared.md:6]:
```suggestion
You have two independent environments (`main` & `dev`) in the cloud and have corresponding Git branches with your Amplify backend infrastructure code on Git. Suppose all team members want to work on the same Amplify project and push backend related changes to the same `dev` environment to test their changes. Each team member would run the following:
```

Comment (tjleing) [reply docs/cli/teams/shared.md:6]: Git is correct (as opposed to git), as is Amplify vs. amplify.  Let's keep `main` and `dev` as opposed to main and dev as well.

**Pull request merged**